packages/logging-winston/README.md

```diff
@@ -1,44 +1,79 @@
 # @google-cloud/logging-winston
-> Stackdriver Logging transport for Winston
+> Stackdriver Logging transport for [Winston][winston]
 
-This module provides a easy to use, higher level layer for working with Stackdriver Logging that works with Winston. Simply attach this as a transport to your existing Winston loggers.
+This module provides an easy to use, higher-level layer for working with Stackdriver Logging, compatible with Winston. Simply attach this as a transport to your existing Winston loggers.
 
-If you want have a lower level access to the Stackdriver Logging API, check out [@google-cloud/logging](https://www.npmjs.com/package/@google-cloud/logging).
+For lower-level access to the Stackdriver Logging API, see [@google-cloud/logging][@google-cloud/logging].
 
-> *This module is experimental, and should be used by early adopters. This module uses APIs that may be undocumented and subject to change without notice.* 
+> *This module is experimental and should be used by early adopters. This module uses APIs that may be undocumented and subject to change without notice.*
 
 ``` sh
 $ npm install --save @google-cloud/logging-winston
 ```
 ``` js
 var winston = require('winston');
-var StackdriverTransport = require('@google-cloud/logging-winston');
-var logger = new winston.Logger({
-  transports: [ new winston.transports.StackdriverLogging() ]
+var stackdriverTransport = require('@google-cloud/logging-winston');
+
+winston.add(stackdriverTransport, {
+  projectId: 'grape-spaceship-123',
+  keyFilename: '/path/to/keyfile.json',
+  level: 'warning', // log at 'warning' and above
+  resource: {
+    type: 'global'
+  }
 });
+
 logger.error('warp nacelles offline');
 logger.verbose('sheilds at 99%');
 ```
 
-### Optional Configuration
+## Authentication
+
+It's incredibly easy to get authenticated and start using Google's APIs. You can set your credentials on a global basis as well as on a per-API basis. See each individual API section below to see how you can auth on a per-API-basis. This is useful if you want to use different accounts for different Google Cloud services.
 
-If your app is running on Google Cloud Platform, we handle all configuation & authentication for you.  We also auto-detect the appropriate [resource descriptor](todo://add/link) to report the log entries against.
+### On Google Compute Engine
 
-If you are running your application in a different environment, for example, locally, on-premise, or on another cloud provider, you may need to provide some configuration.
+If you are running this client on Google Compute Engine, we handle authentication for you with no configuration. You just need to make sure that when you [set up the GCE instance][gce-how-to], you add the correct scopes for the APIs you want to access.
```

Comment:
Oops, I didn't mean to squash that. In fact, I changed it from GCE to GCP across all the readmes in a PR today.

When you create an instance through the console, one of the options presented is setting the scopes. It's at that time I expect the user to understand the connection. But thanks for pointing out that it's not required here, I'll remove it. And suggestions for how to improve the docs are welcome as well, as the other APIs do require a user to enable the correct scopes.